test/xvfb_spec.js

```diff
@@ -37,4 +37,28 @@ describe('xvfb', function () {
       expect(process.env.DISPLAY).to.be.undefined
     })
   })
+
+  context('errors', () => {
+    let xvfb
+    it('logs error only once', (done) => {
+      const stub = sinon.stub()
+      .callsFake(() => {
+        // first call for initial process start
+        // second call for error
+        // no third call, since onError should only be called once
+        stub.callsFake(() => {
+          expect(stub.callCount, 'onError should only call the cb once').eq(2)
```

Comment:
is this an accurate message?

Reply:
Seems accurate? What were you thinking it should say

Reply:
`expect(stub.callCount) ... eq(2)` right? But the message says "onError should only call the cb once" So I cannot square 2 with once. Especially since we removed the "once" completely

Reply:
first call is the xvfb starting up so i have access to the child process
then the second call is called with the error